astropy/units/tests/test_units.py

```diff
@@ -720,21 +720,29 @@ def test_pickle_unrecognized_unit():
 
 
 @pytest.mark.parametrize(
-    "name",
+    "name,message",
     [
-        pytest.param("h", id="simple_conflict"),
-        pytest.param("ʰ", id="NFKC_normalization"),
+        pytest.param(
+            "h",
+            r"^the namespace already uses the name 'h' for Unit\(\"h\"\)$",
```

Comment:
Having to escape the quotes in this regex annoyed me so much that it made me open a pull request to change the quotes `UnitBase.__repr__()` writes out. But clearly that change will not happen in v7.2.